plugins/modules/sysctl.py

```diff
@@ -56,6 +56,34 @@
             - Verify token value with the sysctl command and set with -w if necessary
         type: bool
         default: 'no'
+    attributes:
+        description:
+            - This option is not used.
+    group:
+        description:
+            - This option is not used.
+    owner:
+        description:
+            - This option is not used.
+    selevel:
+        description:
+            - This option is not used.
+    serole:
+        description:
+            - This option is not used.
+    setype:
+        description:
+            - This option is not used.
+    seuser:
+        description:
+            - This option is not used.
```

Comment:
Unlike `unsafe_writes`, which is about the *process* of modifying a file, all file properties (`mode`, `group`, `owner`, `attributes`...) are supported by the call to `module.set_fs_attributes_if_different()`, which is about file's *static* metadata. So, all these options are fully supported, as well as `mode`, and should not be overridden this way here.
```suggestion
```

Reply:
done! :)